Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<ul><li>Under the hood, ArviZ turns the trace into <code>InferenceData</code> each it's called. So, when you're gonna use several ArviZ functions for the same trace, it's usually more efficient to cast the trace to ID after sampling under the model context: <code>idata_6_1 = az.from_pymc3(m_6_1_trace)</code>. And then you can pass <code>idata_6_1</code> instead of the trace to ArviZ.</li><li>Also, no need to wrap <code>mu</code> in a <code>Deterministic</code>, as it's not used after that.</li><li>Maybe add <code>round_to=2</code>  in the summary, to mirror the book.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>